internal/task/task.go

```diff
@@ -0,0 +1,187 @@
+package task
+
+import (
+	"errors"
+	"github.com/ZnNr/go-todo/internal/nextdate"
+	"github.com/ZnNr/go-todo/internal/settings"
+	"strconv"
+	"time"
+)
+
+var (
+	ErrRequireTitle = errors.New("require task title")
+	ErrNotFoundTask = errors.New("not found task")
+)
+
+// Task Структура представляет собой модель задачи
+type Task struct {
+	Id      string `json:"id"`
+	Date    string `json:"date"`
+	Title   string `json:"title"`
+	Comment string `json:"comment"`
+	Repeat  string `json:"repeat"`
+}
+
+// TaskList Структура представляет собой список задач
+type TaskList struct {
```

Comment:
Обычно если экспортируемое название содержит название пакета, то его убирают, чтобы потом импорт выглядел лучше (где-то в другом пакете будет `task.TaskList`, а лучше `task.List`)
```suggestion
type List struct {
```